### ISSUE TITLE
README says daytimes is optional, but if you leave it off...

### ISSUE THREAD

**Issue by jfrux** [benleb/ad-automoli#5]
``` 
 File "/usr/local/lib/python3.7/site-packages/appdaemon/app_management.py", line 145, in initialize_app
    await utils.run_in_executor(self, init)
  File "/usr/local/lib/python3.7/site-packages/appdaemon/utils.py", line 277, in run_in_executor
    response = future.result()
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/conf/apps/automoli/automoli.py", line 115, in initialize
    for idx, daytime in enumerate(daytimes):
TypeError: 'NoneType' object is not iterable
```

Please forgive me if I'm missing something!

**Comment by benleb** [comment]
oh, I will have a look... can you show me your config please? (just for the affected app, not all appdaemon config)
does it work if you configure some daytimes?

**Comment by jfrux** [comment]
To be honest, it's kind of petty honestly...
If you put "daytime: []" it's fine... but when I think about "optional" I think you can exclude it completely.

But it's not a big deal as it is, but it definitely appears to be requiring the key to exist in array form for it to work.

**Comment by benleb** [comment]
Agreed, it also should work without the key at all... I will fix this today or tomorrow ✌️

**Comment by jfrux** [comment]
But dude! @benleb GREAT WORK! Thank you for making this.
Will contribute soon!

**Comment by benleb** [comment]
what a stupid bug, shame on me...

fixed for python 3.8 (using the new walrus operator). fix for older versions tomorrow...
